starport/cmd/relayer_configure.go

```diff
@@ -13,31 +13,31 @@ import (
 )
 
 const (
-	flagAdvanced         = "advanced"
-	flagSourceRPC        = "source-rpc"
-	flagTargetRPC        = "target-rpc"
-	flagSourceFaucet     = "source-faucet"
-	flagTargetFaucet     = "target-faucet"
-	flagSourcePort       = "source-port"
-	flagSourceVersion    = "source-version"
-	flagTargetPort       = "target-port"
-	flagTargetVersion    = "target-version"
-	flagSourceGasPrice   = "source-gasprice"
-	flagTargetGasPrice   = "target-gasprice"
-	flagSourceAddrPrefix = "source-prefix"
-	flagTargetAddrPrefix = "target-prefix"
-	flagOrdered          = "ordered"
+	flagAdvanced        = "advanced"
+	flagSourceRPC       = "source-rpc"
+	flagTargetRPC       = "target-rpc"
+	flagSourceFaucet    = "source-faucet"
+	flagTargetFaucet    = "target-faucet"
+	flagSourcePort      = "source-port"
+	flagSourceVersion   = "source-version"
+	flagTargetPort      = "target-port"
+	flagTargetVersion   = "target-version"
+	flagSourceGasPrice  = "source-gasprice"
+	flagTargetGasPrice  = "target-gasprice"
+	flagSourceKeyPrefix = "source-prefix"
```

Comment:
I was going to change Addr with Address (we usually use Addr as a short cut for urls) then replaced with Key anyway. We can change this again later if you want but probabbly just the flags because it takes too much time otherwise. 😄

Reply:
i'llchange it to AddressPrefix everyehwere so it matches what we use in starport